backends/portage/portageBackend.py

```
@@ -2008,6 +2008,8 @@ def repo_enable(self, repoid, enable):
                            "gentoo repository can't be disabled")
             return
 
+        if not re.match(r'^[a-zA-Z0-9][a-zA-Z0-9._-]*$', repoid):
+            raise ValueError("Invalid repository identifier: {}".format(repoid))
```

Ruff (0.15.12)

[warning] Use f-string instead of `format` call

Convert to f-string

(UP032)